dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -450,11 +463,14 @@ def _prep_input_sets(self, inputs):
         inputs = list(inputs)
 
         for i, input in enumerate(inputs):
-            # Transforming any convertible datatype to TensorList (DataNodeDebugs are already unpacked).
+            # Transforming any convertible datatype to
+            # TensorList (DataNodeDebugs are already unpacked).
             # Additionally accepting input sets, but only as list of TensorList.
-            if not isinstance(input, (_tensors.TensorListCPU, _tensors.TensorListGPU)) and \
-                    not (isinstance(input, list) and
-                         all([isinstance(elem, (_tensors.TensorListCPU, _tensors.TensorListGPU)) for elem in input])):
+            if (not isinstance(input, (_tensors.TensorListCPU, _tensors.TensorListGPU))
+                    and not (isinstance(input, list) and all([
+                        isinstance(elem, (_tensors.TensorListCPU, _tensors.TensorListGPU))
+                        for elem in input
+                    ]))):
```

Comment:
The ever increasing indentation suggests that those are somehow nested within what's in the previous lines. I had to count parentheses to get what's going on here.
```suggestion
            if (not isinstance(input, (_tensors.TensorListCPU, _tensors.TensorListGPU))
                and not (isinstance(input, list)
                and all([
                    isinstance(elem, (_tensors.TensorListCPU, _tensors.TensorListGPU))
                    for elem in input
                ]))):
```

Reply:
Your suggestion will not pass the linter.

Reply:
Done with 1.